src/overlays/actors/ovl_En_Nutsball/z_en_nutsball.h

```diff
@@ -7,7 +7,9 @@ struct EnNutsball;
 
 typedef struct EnNutsball {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x50];
+    /* 0x144 */ s16 timer;
+    /* 0x146 */ UNK_TYPE2 unk146;
```

Comment:
```suggestion
    /* 0x146 */ s16 unk146;
```
Looking at the code I think this could be an s16. This is also being compared against the timer, so it could be tracking the duration of something?

Reply:
Looks to me like the OC doesn't get updated until the timer is below this variable, so decided to go with startTimeOC.